paradise_papers_search/fetch_api/models/helpers.py

```diff
@@ -59,8 +59,10 @@ def fetch_node_details(node_info):
     node_id         = node_info['node_id']
     node            = MODEL_ENTITIES[node_type].nodes.get(node_id=node_id)
     node_details    = node.serialize
+    node_details['node_connections'] = []
```

Comment:
I think that it work the same without this assignation 🤔 
What you think?

Reply:
I just want to always return a empty array if there is not connections, for convenience when consuming the API